experiments/34x25/common_args.m

```diff
@@ -27,7 +27,7 @@
 
   cargs = {  'parallel', false, 'debug', 1:10, 'ac.debug', 1:10, 'p.debug', 1:10, ...
              'ac.randState', 2,   'p.randState', 2, ...
-             'distn',   {'normem2'},     'mu',        0,  'spacing', struct('mean', 2.318, 'maxdiffpct', 0.5), ...%2245), ...
+             'distn',   {'normem2'},     'mu',        0, ... % 'spacing', struct('mean', 2.318, 'maxdiffpct', 0.5), ...%2245), ...
```

Comment:
As I mentioned, this didn't have to be committed... it's fine, but keep in mind in the future that it's rare to `git commit -a`.